knexfile.js

```diff
@@ -0,0 +1,26 @@
+const knex = require('knex');
+
+module.exports = {
+
+    development: {
+
+        client: 'mysql',
+
+        connection: {
+            host : 'localhost:3306',
+            user: 'root',
+            password: 'testdb1',
```

Comment:
Hey bud. Thanks for all the hard work today. I know this is just the username and password for a disposable test db. After having credentials get leaked in my build week, and all the emails that bots sent us, I am really cautious about putting anything close to a username and pass in the source code. Maybe just make these blank before a push, and populate them locally. Then you could leave a comment above that they need to be filled before connecting

```
// instert actual user name and pass, or env vars to connect to db
username: '',
password: '',